app/src/androidTest/java/kommunicate/io/sample/ConversationListTest.kt

```diff
@@ -7,7 +7,7 @@ import androidx.test.espresso.Espresso.onView
 import androidx.test.espresso.matcher.ViewMatchers.isRoot
 import androidx.test.espresso.matcher.ViewMatchers.withId
 import androidx.test.ext.junit.runners.AndroidJUnit4
-import com.applozic.mobicomkit.api.account.register.RegistrationResponse
+import dev.kommunicate.devkit.api.account.register.RegistrationResponse
```

Comment:
Why package name is used as dev.kommunicate instaed of io.kommunicate ? We are not owning .dev domain and someone might register it in feature